import_logs.py

```diff
@@ -2367,12 +2331,6 @@ def filtered_line(line, reason):
             if not line: break
             lineno = lineno + 1
 
-            try:
-                line = line.decode(config.options.encoding)
-            except UnicodeDecodeError:
-                invalid_line(line, 'invalid encoding')
-                continue
-
```

Comment:
I'd rather keep that code and open the files in binary mode instead, which lets us keep the `encoding` option and the `invalid encoding` statistic.